translations/compfest-11-penyisihan/compfest-11-penyisihan-junior-pertarungan-terakhir.html

```diff
@@ -0,0 +1,141 @@
+<!-- Title : The Last Battle -->
+
+<h3>Description</h3>
+
+<p style="text-align: justify;">
+	Mr. Chanek is playing his favorite video game. In the game, he has $N$ heroes standing in order
+	from the first hero on the left to the $N$-th hero on the right. The $i$-th hero has a power level of $A_i$.

```

Comment:
deskripsi aslinya pakai "Pahlawan i", bukan "Pahlawan ke-i"

```suggestion
	from the first hero on the left to the $N$-th hero on the right. Hero $i$ has a power level of $A_i$.

```